idea-plugin/src/main/kotlin/arrow/meta/ide/plugins/proofs/coercions/implicit/implicitcoercion.kt

```diff
@@ -17,21 +18,23 @@ import org.jetbrains.kotlin.psi.KtSafeQualifiedExpression
 val IdeMetaPlugin.localImplicitCoercion: ExtensionPhase
   get() = addLocalInspection(
     inspection = implicitCoercion,
-    level = HighlightDisplayLevel.WARNING,
-    groupPath = ProofPath + arrayOf("Coercion")
+    groupPath = ProofPath + arrayOf(MetaIdeBundle.message("proofs.coercion")),
+    groupDisplayName = MetaIdeBundle.message("proofs.coercion"),
+    level = HighlightDisplayLevel.WARNING
   )
 
-const val IMPLICIT_COERCION_INSPECTION_ID = "Make_coercion_implicit"
+const val IMPLICIT_COERCION_INSPECTION_ID = "CoercionImplicit"
 
 val IdeMetaPlugin.implicitCoercion: AbstractApplicabilityBasedInspection<KtDotQualifiedExpression>
   get() = applicableInspection(
     defaultFixText = IMPLICIT_COERCION_INSPECTION_ID,
+    staticDescription = MetaIdeBundle.message("proofs.coercions.inspection.implicit.static.description"),
     inspectionHighlightType = { ProblemHighlightType.WARNING },
     kClass = KtDotQualifiedExpression::class.java,
     inspectionText = { ktCall: KtDotQualifiedExpression ->
       // TODO: research ways to display this nicely and align it with [arrow.meta.ide.plugins.proofs.markers.CoercionKt.coerceProofLineMarker]
-      val coercionMessage = ktCall.ctx()?.coercionProofMessage(ktCall)
-      "Expression: ${ktCall.text} can be replaced for only its receiver because there is a $coercionMessage"
+      val coercionMessage = ktCall.ctx()?.coercionProofMessage(ktCall).orEmpty()
```

Comment:
When can context be null on this scenario? Looks like the empty state should never happen since it'd be a UX error.